packages/core/src/utils/JsonTransformer.ts

```diff
@@ -1,15 +1,41 @@
+import type { Validate } from 'class-validator'
+
 import { instanceToPlain, plainToInstance, instanceToInstance } from 'class-transformer'
 
+import { ClassValidationError } from '../error/ClassValidationError'
+
+import { MessageValidator } from './MessageValidator'
+
+interface Validate {
+  validate?: boolean
+}
+
 export class JsonTransformer {
   public static toJSON<T>(classInstance: T) {
     return instanceToPlain(classInstance, {
       exposeDefaultValues: true,
     })
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  public static fromJSON<T>(json: any, Class: { new (...args: any[]): T }): T {
-    return plainToInstance(Class, json, { exposeDefaultValues: true })
+  public static fromJSON<T>(
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    json: any,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    Class: { new (...args: any[]): T },
+    { validate = true }: Validate = {}
```

Comment:
```suggestion
    { validate = true }: Validate
```

What is the empty object assignment there? It would collide with the `{validate: true}` and one would take precedence (not sure which one).

Reply:
`{ validate = true }: Validate` does not work with `    TypeError: Cannot read properties of undefined (reading 'validate')` whereas ` { validate = true }: Validate = { validate: true }` gives the same (correct) behaviour as `    { validate = true }: Validate = {}`. not 100% sure what shorthand quirk of typescript that is exactly - but it works😅

Reply:
Hmmm, I will take a look at this later as this is not really how it is supposed to work haha.